api/src/controllers/middleware.ts

```diff
@@ -100,23 +110,40 @@ function authorize(scope: Scope): AuthMiddleware {
  * Returns middleware that will ensure the client accessing is authorized and will continue matching routes if the are not.
  * @returns authorizeAndFallThrough middleware.
  */
-function authorizeAndFallThrough(scope: Scope): AuthMiddleware {
+function authorizeAndFallThrough(scope: Scope): ErrHandledMiddleware {
     // Check that the caller has the proper scopes
     const checkAuthorization = jwtAuthz([scope], { failWithError: true });
 
     // Catch errors thrown off by jwtAuthz and fallthrough
     const errorHandler = (err: Error, req: Request, res: Response, next: NextFunction) => {
         // Fallthrough to the next route in the router
-        next('route');
+        next('route'); // TODO only convert error when thrown by jwtAuthz
```

Comment:
How the heck do I check that the error was thrown only by jwtAuthz? @armiantos

Reply:
Can we do
```ts
if (err instance of <JwtErrorAuthType>)
```
I'll have to test out on my machine to find out what the error type is though. Leave this to me, I'll get it done by tonight

Reply:
That's what I wanted to do, but they don't use a custom error. They just return a generic object.

Reply:
Yea, I looked at [their source code](https://github.com/auth0/express-jwt-authz/blob/4591867995181a4568fb38f9c4b00c658116a815/lib/index.js#L12-L18) and it seems like they're just returning an object with 3 properties (not even an Error object). 

So I think best we can do is `if (err.message === 'Insufficient scope')` because:
* this lets us stick to our es5 `Error` type
* won't require nasty `any` castings

Reply:
Cool, agreed. I'll make the change and then would like another review.

Reply:
Just pushed the fix